custom_components/viaris/const.py

```diff
@@ -53,6 +53,7 @@
 LIMIT_POWER_KEY = "limit_power"
 SELECTOR_POWER_KEY = "selector_power"
 MQTT_URL_KEY = "mqtt_url"
+MAX_CONFIGURED_POWER_KEY = "max_configured_power"
```

Comment:
CURRENT_MAX_POWER_KEY = "current_max_power" instead of MAX_CONFIGURED_POWER_KEY = "max_configured_power"